mmcv/runner/hooks/logger/pavi.py

```diff
@@ -61,7 +65,13 @@ def before_run(self, runner):
             self.init_kwargs = dict()
         self.init_kwargs['task'] = self.run_name
         self.init_kwargs['model'] = runner._model_name
-
+        if runner.meta is not None and 'config_dict' in runner.meta:
+            meta = runner.meta['config_dict'].copy()
+            meta.setdefault('max_iter', runner.max_iters)
+            # non-serializable values are first converted in mmcv.dump to json
+            meta_json = json.loads(mmcv.dump(meta, file_format='json'))
```

Comment:
The variable name `meta_json` is improper. It is still a dict instead of json string.